sdk/go/pulumi/run.go

```diff
@@ -35,8 +37,13 @@ type RunOption func(*RunInfo)
 // If the program fails, the process will be terminated and the function will not return.
 func Run(body RunFunc, opts ...RunOption) {
 	if err := RunErr(body, opts...); err != nil {
-		fmt.Fprintf(os.Stderr, "error: program failed: %v\n", err)
-		os.Exit(1)
+		if err != ErrPlugins {
+			fmt.Fprintf(os.Stderr, "error: program failed: %v\n", err)
+			os.Exit(1)
+		}
+
+		printRequiredPlugins()
+		os.Exit(0)
 	}
```

Comment:
Total nit - and no change needed - but I was surprised this wasn't inverted (`if err == ErrPlugins` {`).